lib/rules/jsx-closing-tag-location.js

```diff
@@ -18,9 +18,11 @@ const report = require('../util/report');
 // ------------------------------------------------------------------------------
 
 const messages = {
-  onOwnLine: 'Closing tag of a multiline JSX expression must be on its own line.',
+  onOwnLine:
+    'Closing tag of a multiline JSX expression must be on its own line.',
```

Comment:
```suggestion
  onOwnLine: 'Closing tag of a multiline JSX expression must be on its own line.',
```
please don't run prettier on projects not configured to use it :-)